releases/release-1.13/README.md

```diff
@@ -0,0 +1,139 @@
+* Unlike the previous cycles key deliverables, including the final release, do **not** fall on Tuesdays due to shortened release cycle.
+* The release cycle length is approximately only **10 weeks** long.
+
+## Timeline
+
+
+| **What** | **Who** | **Oct** | **Nov** | **Dec** | **Jan** | **DEV WEEK** | **TEST GATES** |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| Start of Release Cycle | Lead | 1| | | | week 1 | |
+| Finalize Schedule | Lead | 5 | | | | | |
+| Begin collecting planned work from SIGs | Lead, Enhancements Lead | 8| | | | week 2 | |
+| Begin weekly release team meetings | Lead | 8 | | | | | |
+| Begin weekly status reports at Community | Lead, Shadow | 11 | | | | | |
+| Finalize Release Team | Lead | 12 | | | | | |
+| Start Release Notes Draft | Release Notes Lead | 16 | || | week 3 | |
+| Clean up enhancements repo | Enhancements Lead | 16 | | | | | |
+| "Enhancements Freeze" begins (EOD PST) | Enhancements Lead | 23 | || | week 4 | |
+| 1.13.0-alpha.1 release | Branch Manager | 23 | | | | | [master-blocking], [master-upgrade] |
+| Blog post: what we're working on for 1.13 | Communications | | 2 | | | week 5 | |
+| All release-1.9 CI jobs/testgrid removed | Test Infra Lead | | 6 | | | week 6 | |
+| Create 'release-1.13' branch and begin daily branch | Branch Manager | | 6 | | | | |
+| v1.13.0-alpha.0 | Branch Manager | | 6 | | | | |
+| 1.13.0-beta.0 released from branch | Branch Manager | | 6 | | | | [master-blocking], [master-upgrade] |
+| Docs deadline - Open placeholder PRs | Docs Lead | | 8 | | | | |
+| All release-1.13 CI jobs/testgrid created | Test Infra Lead | | 9 | | | | |
+| Begin Code Slush | Bot, Lead | | 9 | | | week 6 | |
+| Begin MWF Burndown meetings | Lead | | 12 | | | week 7 | |
+| All Issues & PRs must have complete labels | Bug Triage | | 12 | | | | |
+| Begin code and test freeze (EOD PST) | Bot, Lead | | 15 | | | week 7 | [1.13-blocking], [master-blocking], [master-upgrade] |
+| Begin pruning | Lead and release team | | 16 | | | | |
+| 1.13.0-beta.1 released from branch | Branch Manager | | 16 | || | [1.13-blocking], [master-blocking], [master-upgrade] |
+| Begin M-F Burndown meetings | Lead | | 26 | | | week 8 | |
+| Unapproved/non-critical Issues/PRs get kicked out | Bot, Bug Triage | | 19 | | | | |
+| Docs deadline - PRs ready for review | Docs Lead | | 19 | | | | |
+| 1.13.0-beta.2 released from branch | Branch Manager | | 21 | | | | [1.13-blocking], [master-blocking], [master-upgrade] |
+| Docs complete - All PRs reviewed and ready to merge | Docs Lead | | 27 | | | week 9 | |
+| 1.13.0-rc.1 released from branch | Branch Manager | | 27 | | | | [1.13-blocking], [master-blocking], [master-upgrade] |
+| End of code freeze (EOD PST): Master branch re-opens for 1.13 | Bot, Branch Manager, Lead | | 28 | | | | |
```

Comment:
I'm concerned that 48 hours for code thaw-to-final-cherrypick is about 24 hours too short.  We need enough time for folks in multiple geos to review/authorize any trailing cherrypicks or fixes.

Reply:
I agree Josh that we might need a day more but I would like to aim for a shorter thaw and CP period and extend it in the worst case. With the current schedule there is almost a week of buffer(before Kubecon), thus giving us room to extend thaw by a day if need be.